memconfig.json

```diff
@@ -2345,6 +2356,157 @@
           }
         ],
         "description": "Collection of tests for the Github driver"
+      },
+      {
+        "name": "PullRequestReviewRequestEvent",
+        "events": [],
+        "fields": [
+          {
+            "name": "review_request",
+            "type": "Ref",
+            "ofType": "PullRequest"
+          }
+        ]
+      },
+      {
+        "name": "PullRequestReviewRequest",
+        "events": [],
+        "actions": []
+      },
+      {
+        "name": "Organization",
+        "fields": [
```

Comment:
I see what you mean here that `Organization` is nearly identical to `User`...

Reply:
But my gut tells me some not-DRY schema is better than trying to overgeneralize and couple users with orgs